docs/cluster-wide.md

```diff
@@ -0,0 +1,163 @@
+
+Let’s say you have a Namespace in your Kubernetes cluster called `percona-db-1`.
+
+1. Create your `percona-db-1` namespace (if it doesn't yet exist) as follows:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl create namespace percona-db-1
+    ```
+
+2. Deploy the Operator:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/operator.yaml -n percona-db-1
+    ```
+
+3. Once Operator is up and running, deploy the database cluster itself:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cr.yaml -n percona-db-1
+    ```
+
+You can deploy multiple clusters in this namespace.
+
+### Add more namespaces
+
+What if there is a need to deploy clusters in another namespace? The solution for namespace-scope deployment is to have more than one Operator. We will use the `percona-db-2` namespace as an example.
+
+1. Create your `percona-db-2` namespace (if it doesn't yet exist) as follows:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl create namespace percona-db-2
+    ```
+
+2. Deploy the Operator:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/operator.yaml -n percona-db-2
+    ```
+
+3. Once Operator is up and running deploy the database cluster itself:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cr.yaml -n percona-db-2
+    ```
+
+    !!! note
+
+        Cluster names may be the same in different namespaces.
+
+## Install the Operator cluster-wide
+
+Sometimes it is more convenient to have one Operator watching for
+Percona Distribution for PostgreSQL custom resources in several namespaces.
+
+We recommend running Percona Operator for PostgreSQL in a traditional way,
+limited to a specific namespace, to limit the blast radius. But it is possible
+to run it in so-called *cluster-wide* mode, one Operator watching several
+namespaces, if needed:
+
+![image](assets/images/cluster-wide-2.svg)
+
+To use the Operator in such cluster-wide mode, you should install it with a
+different set of configuration YAML files, which are available in the deploy
+folder and have filenames with a special `cw-` prefix: e.g.
+`deploy/cw-bundle.yaml`.
+
+Using `cw-*` manifests will create a new namespace (`pg-operator` by default),
+deploy the Operator into it. Deployed this way, the Operator will start
+monitoring all namespaces for `PerconaPGCluster` custom resources.
+
+1. Clone `percona-postgresql-operator` repository:
+
+    ``` {.bash data-prompt="$" }
+    $ git clone -b v{{ release }} https://github.com/percona/percona-postgresql-operator
+    $ cd percona-postgresql-operator
+    ```
+
+2. Deploy the bundle. It will create the `pg-operator` namespace, and deploy the
+    Operator into it.
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cw-bundle.yaml –server-side
+    ```
+
+    !!! note
+
+        If you want to change the namespace, please edit
```

Comment:
I discussed it with Ege yesterday, and Ege will improve it in the same way as we have for all other operators. Please wait for changes.